### ISSUE TITLE
AssertionError: failed lofting when running radiation-daysim for a building with no height above ground

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#1628]
This case just came up in conjunction with issue #1625. There was a building with

- floors_ag = 0
- floors_bg = 1
- height_ag = 0
- height_bg = 4.5

in the `district.shp` file. This created an error in the `geometry_generator.py` script:

```
Processing district building Z1947
Traceback (most recent call last):
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\1224-improve-runtime-for-solar-collectors\Scripts\cea-script.py", line 11, in <module>
    load_entry_point('cityenergyanalyst', 'console_scripts', 'cea')()
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\interfaces\cli\cli.py", line 63, in main
    script_module.main(config)
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\resources\radiation_daysim\radiation_main.py", line 213, in main
    settings)
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\resources\radiation_daysim\geometry_generator.py", line 372, in geometry_main
    nfloor_col="floors_ag")
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\resources\radiation_daysim\geometry_generator.py", line 170, in building2d23d
    building_solid = calc_solid(face_footprint, range_floors, flr2flr_height)
  File "c:\users\darthoma\documents\github\cityenergyanalyst\cea\resources\radiation_daysim\geometry_generator.py", line 297, in calc_solid
    vertical_shell = construct.make_loft(moved_face_list)
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\1224-improve-runtime-for-solar-collectors\lib\site-packages\py4design\py3dmodel\construct.py", line 1175, in make_loft
    loft = Construct.make_loft(wire_list, ruled = rule_face, tolerance = tolerance )
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\1224-improve-runtime-for-solar-collectors\lib\site-packages\py4design\py3dmodel\OCCUtils\Construct.py", line 423, in make_loft
    with assert_isdone(sections, 'failed lofting'):
  File "C:\Users\darthoma\AppData\Local\conda\conda\envs\1224-improve-runtime-for-solar-collectors\lib\site-packages\py4design\py3dmodel\OCCUtils\Common.py", line 65, in __enter__
    raise AssertionError(self.error_statement)
AssertionError: failed lofting
```

The radiation-daysim script needs to be updated to handle this special case. I propose to add such a building to the `reference-case-open/baseline` so that it is included with the tests. Buildings with no height above ground should be excluded from the radiation analysis.

@JIMENOFONSECA do you want look into this?

**Comment by JIMENOFONSECA** [comment]
@daren-thomas I suggest to create an exception so the program stops if this is the case.

**Comment by JIMENOFONSECA** [comment]
This will be solved in PR #1671
